src/content/docs/en/guides/routing.mdx

```diff
@@ -459,41 +459,6 @@ const { page } = Astro.props;
 {page.url.last ? <a href={page.url.last}>Last</a> : null}
 ```
 
-
-#### Complete API reference
```

Comment:
You raise a good point. We lose the convenience of glancing at all of the available properties, & having to navigate through the wall of text in the API Reference section... I agree that most of the properties are self explanatory. I can imagine that the some users may find it inconvenient to scroll all that just to see the available properties.

Considering the repercussions, keeping that block there seems warranted even though it adds to the maintenance costs.... Although, it still irks me. 😓 Surely, we can do better....

In hindsight, I realize that the actual purpose of that code block there was to serve as an _At a glance_ section to showcase all of the supported properties by `Page`. 

I was going to suggest that we simplify the type to a block which only mentions all of the available properties, but I realize that it would be much more work than simply copy-pasting the updated type from the codebase.

Should I revert back to its original state then?

Reply:
I made a commit with an example of what I think could be a helpful structure to work from. What do you think?

Reply:
though, we did lose this text which might go in the block itself for `data`

> array containing the page’s slice of data that you passed to the paginate() function

Reply:
It seems we are coming back to https://github.com/withastro/docs/issues/8929#issuecomment-2255932662 😅 So, it's probably better to leave the block here. And, I agree with Sarah, we could replace `result` which is not descriptive enough with the sentence quoted!

Reply:
I think that adding the description for `data` property in the block would work perfectly fine.

Reply:
OK, I think I'm good with where things are here!